tests/foreman/api/test_subnet.py

```diff
@@ -32,119 +40,170 @@
 class ParameterizedSubnetTestCase(APITestCase):
     """Implements parametrized subnet tests in API"""
 
-    @stubbed()
+    @tier1
+    def test_positive_create_with_parameter(self):
+        """Subnet can be created along with parameters
+        :id: ec581cb5-8c48-4b9c-b536-302c0b7ec30f
+        :steps: Create Subnet with parameter that has single key and single
+            value
+        :expectedresults: The Subnet is created with parameter
+        """
+        parameter = [
+            {'name': gen_string('alpha'), 'value': gen_string('alpha')}
+        ]
+        subnet = entities.Subnet(
+            subnet_parameters_attributes=parameter
+        ).create()
+        self.assertEqual(
+            subnet.subnet_parameters_attributes[0]['name'],
+            parameter[0]['name']
+        )
+        self.assertEqual(
+            subnet.subnet_parameters_attributes[0]['value'],
+            parameter[0]['value']
+        )
+
     @tier1
     def test_positive_add_parameter(self):
         """Parameters can be created in subnet
-
         :id: c1dae6f4-45b1-45db-8529-d7918e41a99b
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter with single key and single value
-
         :expectedresults: The parameter should be created in subnet
         """
+        subnet = entities.Subnet().create()
+        for name in generate_strings_list(exclude_types=['html']):
+            with self.subTest(name):
+                value = gen_string('utf8')
+                subnet_param = entities.Parameter(
+                    subnet=subnet.id,
+                    name=name,
+                    value=value
+                ).create()
+                self.assertEqual(subnet_param.name, name)
+                self.assertEqual(subnet_param.value, value)
 
-    @stubbed()
     @tier1
     def test_positive_add_parameter_with_multiple_values(self):
         """Subnet parameters can be created with multiple values
-
         :id: b3de6f96-7c39-4c44-b91c-a6d141f5dd6a
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having single key and multiple values
                 separated with comma
-
         :expectedresults: The parameter with multiple values should be saved
             in subnet
         """
+        subnet = entities.Subnet().create()
+        name = gen_string('alpha')
+        values = ', '.join(generate_strings_list(exclude_types=['html']))
```

Comment:
Once more, why does that value called as multiple?
Is 'test, test2 ,test3' not equal to 'test' or to 'test/test2/test3'  or to 'erdtgdsrfg,ewrtg,wsert' from logic perspective? It is still singular value. I think that test case is invalid

Reply:
Here I just want to test that can Value contains values separated by comma. I gave you an example of the address of a location.
E.g:
```
Satellite Street, NY, America
```
Its a very valid case.

Reply:
In fact, the customer who has requested the subnet parameters feature is going to use key value parameters to store the location of subnet/network.

Reply:
I am just removing the word multiple for you.